doc/getting_started/build_sw.md

```diff
@@ -13,12 +13,18 @@ All OpenTitan software is built with [Bazel](https://bazel.build/).
 Additionally, _most_ tests may be run with Bazel too.
 
 ## TL;DR
-To install the correct version of bazel, build, and run all on-host tests you can simply run:
+To install the correct version of bazel, build, and run a single test with Verilator, run:
 
 ```console
-$REPO_TOP/bazelisk.sh test //... --test_tag_filters=-cw310,-verilator --disk_cache=~/bazel_cache
+$REPO_TOP/bazelisk.sh test --test_output=streamed --test_tag_filters=verilator --disk_cache=~/bazel_cache //sw/device/tests:uart_smoketest
```

Comment:
NIT: This invocation is correct but has a trap in it:
The macro `//sw/device/tests:uart_smoketest` will build all its targets including software for the fpga and dv. (No big deal but it takes longer on small non-parallel systems like Jake.) If someone tries to use a similar invocation to build SW for other targets, they might be surprised to see verilator running to support the verilator tests in this macro (which then won't be executed because of the `--test_tag_filter`).

There's a whole set of traps related to how builds and tests are filtered: `--build_tag_filters` and `--build_tests_only` prune targets matched by wildcards, not dependencies so they wont have an effect here and are useful but not as relevant as they might seem and don't offer an answer.

Discovering the platform dependent test's labels is more difficult. They don't appear in our source code and tab completion doesn't work for them, but they're the best way to run single tests and have the build behave as you'd expect:
`$REPO_TOP/bazelisk.sh test --test_output=streamed --disk_cache=~/bazel_cache //sw/device/tests:uart_smoketest_sim_verilator`
This will also help people know those targets exist and how to use them.

Reply:
Okay, sounds good. I'll change the example here to build `uart_smoketest_sim_verilator` and add a little explanation/link to later in the page.